### ISSUE TITLE
Registry tool: registry apply should infer the --parent from values set in configuration

### ISSUE THREAD

**Issue by giteshk** [apigee/registry#781]
1. Created a registry configuration using 
```registry config configurations create config-<gcp-project-name> \
    --registry.address=apigeeregistry.googleapis.com:443 \
    --registry.insecure=0 \
    --registry.project=<gcp-project-name> \
    --registry.location=global
```
2. Tried to run registry apply command 
```
/tmp % registry apply -f openapi-sample.yaml
  INFO[0000] Applying openapi-sample.yaml uid=92b5f12e
 FATAL[0000] Task failed: apply file openapi-sample.yaml error=invalid project name "": must match "^projects/([A-Za-z0-9-.]+)/locations/global$" uid=92b5f12e
```

Expected:
`registry apply` should use the current active configuration and the parent parameter should be optional.

**Comment by timburks** [comment]
Fixed by #783

**Comment by timburks** [comment]
Verified with a test change in #784.
